src/components/relaying-educational/RelaySeriesStepper/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Card, IconButton, SvgIcon } from '@mui/material'
+import Navigator from '@/components/relaying-educational/Navigator'
+import useEducationSeriesStepper, {
+  type EducationSeriesStepperProps,
+} from '@/components/relaying-educational/RelaySeriesStepper/useEducationSeriesStepper'
+import ProgressBar from '@/components/common/ProgressBar'
+import CloseIcon from '@/public/images/common/close.svg'
+import IndexNumber from '@/components/relaying-educational/IndexNumber'
+import css from './styles.module.css'
+
+const RelaySeriesStepper = (props: EducationSeriesStepperProps) => {
+  const { onBack, onNext, activeStep, setStep, onClose, progress } = useEducationSeriesStepper(props)
+  const { steps } = props
+  const currentStep = steps[activeStep]
+
+  return (
+    <Box className={css.wrapper}>
+      <Card className={css.infoCard}>
+        <ProgressBar value={progress} />
+        <Box display="flex" alignItems="center" gap="12px">
+          <IndexNumber value={activeStep + 1} />
+          <h1>{currentStep.title}</h1>
+          <span style={{ flex: '1' }} />
+          <IconButton onClick={onClose}>
+            <SvgIcon component={CloseIcon} inheritViewBox fontSize="small" sx={{ color: 'primary.main' }} />
```

Comment:
This colour does still not match the other modals:

![image](https://user-images.githubusercontent.com/20442784/227157084-b3fa18a5-cbc8-4c84-9338-0bcab06e13d7.png)

![image](https://user-images.githubusercontent.com/20442784/227157104-2fb64e1e-b85b-4ac5-8cfc-96771ae8a05a.png)